tests/eslint-rule-tester.js

```diff
@@ -23,7 +23,7 @@ const defaultConfig = {
     languageOptions: {
         ecmaVersion: 6,
         sourceType: "commonjs",
-        globals: globals.node,
+        globals: { ...globals.es2015, ...globals.node },
```

Comment:
It may be worth using `es2021` here :thinking:

Reply:
better to be same as the `ecmaVersion`. :)